src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -48,21 +65,102 @@ public String getFeedSourceId () {
     @Override
     public void jobLogic () {
         LOG.info("Processing feed for {}", feedVersion.id);
+        feedSource = feedVersion.parentFeedSource();
+        // If there are any feed transformations that apply to the VERSION_CLONE retrieval method, we need to add a
+        // stage to jobFinished (perhaps) to clone the feed version (as long as this feed version has not already been
+        // cloned). Once that feed version is processed, the appropriate transformations will apply themselves.
+        shouldClone = !feedVersion.retrievalMethod.equals(VERSION_CLONE) &&
```

Comment:
Field `shouldClone` should be initialized in the constructor and made `final`.

Reply:
Not a definite answer, but I'd think cloning should be handled by the owner object of the job, so that jobs don't have to deal with retrievalMethod.